src/mjolnir/valhalla_expand_bounding_box.cc

```diff
@@ -0,0 +1,154 @@
+
+  options.add_options()("help,h", "Print this help message.")("version,v",
+                                                              "Print the version of this software.")(
+      "config,c", boost::program_options::value<boost::filesystem::path>(&config_file_path),
+      "Path to the json configuration file.")("inline-config,i",
+                                              boost::program_options::value<std::string>(
+                                                  &inline_config),
+                                              "Inline json config.")
+      // positional arguments
+      ("bounding-box,b", bpo::value<std::string>(&bbox),
+       "Bounding box to expand. The format is lower "
+       "left lng/lat and upper right lng/lat or min_x,min_y,max_x,max_y");
+
+  bpo::positional_options_description pos_options;
+  pos_options.add("bounding-box", 1);
+  bpo::variables_map vm;
+  try {
+    bpo::store(bpo::command_line_parser(argc, argv).options(options).positional(pos_options).run(),
+               vm);
+    bpo::notify(vm);
+  } catch (std::exception& e) {
+    std::cerr << "Unable to parse command line options because: " << e.what() << "\n"
+              << "This is a bug, please report it at " PACKAGE_BUGREPORT << "\n";
+    return EXIT_FAILURE;
+  }
+
+  if (vm.count("help")) {
+    std::cout << options << "\n";
+    return EXIT_SUCCESS;
+  }
+
+  if (vm.count("version")) {
+    std::cout << "valhalla_expand_bounding_box " << VALHALLA_VERSION << "\n";
+    return EXIT_SUCCESS;
+  }
+
+  if (!vm.count("bounding-box")) {
+    std::cout << "You must provide a bounding box to expand.\n";
+    return EXIT_FAILURE;
+  }
+
+  // Read the config file
+  boost::property_tree::ptree pt;
+  if (vm.count("inline-config")) {
+    std::stringstream ss;
+    ss << inline_config;
+    rapidjson::read_json(ss, pt);
+  } else if (vm.count("config") && boost::filesystem::is_regular_file(config_file_path)) {
+    rapidjson::read_json(config_file_path.string(), pt);
+  } else {
+    std::cerr << "Configuration is required\n\n" << options << "\n\n";
+    return EXIT_FAILURE;
+  }
+
+  // configure logging
+  boost::optional<boost::property_tree::ptree&> logging_subtree =
+      pt.get_child_optional("mjolnir.logging");
+  if (logging_subtree) {
+    auto logging_config =
+        valhalla::midgard::ToMap<const boost::property_tree::ptree&,
+                                 std::unordered_map<std::string, std::string>>(logging_subtree.get());
+    valhalla::midgard::logging::Configure(logging_config);
+  }
+
+  std::stringstream ss(bbox);
+  std::vector<float> result;
+
+  while (ss.good()) {
+    std::string substr;
+    getline(ss, substr, ',');
+    result.push_back(std::stof(substr));
+  }
+
+  if (result.size() != 4) {
+    std::cout << "You must provide a valid bounding box to expand.\n";
+    return EXIT_FAILURE;
+  }
+
+  vm::AABB2<vm::PointLL> bb{{result[0], result[1]}, {result[2], result[3]}};
```

Comment:
we should turn this block below into a function on graphreader and then write a small test for it